index.html

```diff
@@ -339,6 +339,22 @@ <h3>Success Criteria</h3>
             </li>
           </ul>
         </div>
+        <div>
+          <h3>Interoperability</h3>
+              <p>The working group will define a syntax for DIDs, a data model for DID Documents, and the requirements for DID methods. Interoperability of implementations for DIDs and DID Documents will be tested by evaluating an implementation's ability to create and parse DIDs and DID Documents that conform to the specification. Interoperability for DID methods will be determined by evaluating each DID method's specification to 
+              determine, at a minimum, </p>
```

Comment:
The first two sentences are good. All the rest either belongs in the DID specification, or the DID Primer. W3C Charters typically talk about what the input documents to a WG are, what is in and out of scope for the WG, and what the expected output is. Almost everything else is left to the WG.

I don't disagree with any of this content, it's just that the more content you add to a W3C Charter, the more there is to debate and block and complain about. All the stuff about "Requirements that a human must manually check" is a huge red flag for many W3C members, so putting that in here is asking for trouble, IMHO.

We should state what the group is expected to do, which is to define requirements on DID Method specifications... we don't need to list those requirements here.

Reply:
I'm okay with what Manu suggests. Joe, if you agree, do you want to make that change?